package.json

```diff
@@ -7,29 +7,29 @@
     "dist"
   ],
   "scripts": {
+    "clean": "del-cli --dot=true \"./dist/**/*\"",
+    "build": "npm run clean && npm run build:compile",
+    "build:compile": "cross-env NODE_ENV=production babel src -d dist --verbose --extensions .js --ignore **/**/*.test.js --source-maps",
```

Comment:
I think we should wrap this glob with globstars in quotation marks so it works across terminals, right?